lib/api/term-meta/functions-admin.php

```diff
@@ -85,23 +87,20 @@ function beans_register_term_meta( array $fields, $taxonomies, $section ) {
  * @ignore
  * @access private
  *
- * @param array $taxonomies Array of taxonomies.
+ * @param array|bool $taxonomies Array of taxonomies or true for all taxonomies.
+ *
  * @return bool
  */
 function _beans_is_admin_term( $taxonomies ) {
-	$taxonomy = beans_get_or_post( 'taxonomy' );
-
-	if ( ! $taxonomy ) {
-		return false;
-	}
-
 	if ( true === $taxonomies ) {
 		return true;
 	}
 
-	if ( in_array( $taxonomy, (array) $taxonomies, true ) ) {
-		return true;
+	$taxonomy = beans_get_or_post( 'taxonomy' );
+
+	if ( empty( $taxonomy ) ) {
+		return false;
 	}
 
-	return false;
+	return ( in_array( $taxonomy, (array) $taxonomies, true ) );
```

Comment:
@iCaspar the outer parentheses are unnecessary. Why? `in_array()` returns a boolean.  So all we have to do is just return what it gives us:

```
return in_array( $taxonomy, (array) $taxonomies, true );
```

Reply:
Doh!